Wei/Utility/UserDefaultsStore.swift

```diff
@@ -13,6 +13,15 @@ protocol UserDefaultsStoreProtocol {
     /// Represents a user's currency
     var currency: String? { get set }
     
+    /// Represents a current network
+    var network: String? { get set }
+    
+    /// Represents a current chainID (used only in private network)
+    var chainID: Int { get set }
```

Comment:
Should we set all of network chain id for helping us to encounter bugs in the future?

Reply:
most of chain ids are defined in Network class but for private net you need to specify which network id to use, and when devs are using private network for debug use, a chain id needs to be stored in UserDefaults